app/src/main/java/eu/kanade/tachiyomi/data/track/anilist/Anilist.kt

```
@@ -167,6 +167,16 @@ class Anilist(id: Long) : TrackService(id) {
         return api.updateLibManga(track)
     }
 
+    override suspend fun delete(track: Track): Track {
+        if (track.library_id == null || track.library_id!! == 0L) {
+            val libManga = api.findLibManga(track, getUsername().toInt())
+                ?: throw Exception("$track not found on user library")
```

Probably safe to just swallow an error in this case.